snapshots/gapminder/2023-09-18/under_five_mortality.xlsx.dvc

```diff
@@ -0,0 +1,30 @@
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Child Mortality Rate (Gapminder)
+    description: |-
+      Gapminder’s historic estimates compiled by Klara Johansson and Mattias Lindgren.
```

Comment:
This description doesn't say what the dataset contains. I'd instead move this to citation_full, and write here a brief line about the content of the dataset.